js_july_23/Users/Sandeep/Homework/Function/homework 18082023.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <script>
+        // Write a function which returns cube-root of number provided in paramter.
+
+        function cubeRoot(n) {
+            let value = n * n * n;
+            return value;
+        }
+
+        let n = 5;
+        document.write("<br> cubeRoot of " + n + " is " + cubeRoot(n) + "<br>");
+        document.write(cubeRoot(4) + "<br>");
+
+    </script>
+
+    <script>
+        // Write a function which returns a power b (3 power 4 => 3 * 3 * 3 * 3), where first parameter is 'a' and second paramter is 'b'. 
+        // E.g., power(a, b) => this returns a * a * a ... b times.
+        function power(a, b) {
+            let result = 1;
+            for (let i = 0; i < b; i++) {
+                result = result * a;
+            }
+            return result;
+        }
+        document.write(power(10, 3) + "<br>");
```

Comment:
I think there is no print commend code written

Reply:
on line number 36 its written